### DISCUSSION TITLE
Adding another location

### DISCUSSION THREAD

**Discussion by damienledger1** [Rory-Sullivan/Dry-Rock#5] [Ideas]
Would it be possible to add more locations to this some mournes locations lowercove possibly, cruit island garron point

**Comment by Rory-Sullivan** [comment]
Hi @damienledger1, sorry for the late response I have been away traveling for the last while. Yeah no problem with adding those extra locations, I should be able to get to it this week. Happy to hear any other suggestions on how to improve the site.

**Reply by damienledger1** [reply]
No other suggestions great resource has got me climbing many times when i thought it wasnt possible

**Reply by Rory-Sullivan** [reply]
The three locations you mentioned above have now been added; Lower Cove, Cruit, and Garron Point.

**Comment by damienledger1** [comment]
Had another idea for dry rock if possible to add a what 3 words location for each crag some crags can be difficult to find i know the purists may have something to say about it but it would be very helpfull also if you wanted to add some new locations that are quite popular atm wexford rocklands quarry and culldaff and malin beg both get good traffic over the summer

**Reply by Rory-Sullivan** [reply]
Hi, I can definitely add those locations, would you mind sending on links to where they are? I'm actually not familiar with Rocklands quarry or Culldaff.

As for adding a what 3 words location, I feel that a simple location pin is not necessarily enough information for how to find a crag and providing a full approach description is out of scope for Dry Rock. However I would be open to providing a link to the UKC (or similar) page for each location. Do you think that would help?

**Comment by damienledger1** [comment]
<http://wiki.climbing.ie/index.php?title=Rocklands>
[Rocklands2.jpg]
Rocklands - Irish Climbing Wiki<http://wiki.climbing.ie/index.php?title=Rocklands>
wiki.climbing.ie<http://wiki.climbing.ie/index.php?title=Rocklands>

<https://www.ukclimbing.com/logbook/crags/dunmore_culdaff-8761/>
[98364.jpeg]
Dunmore, Culdaff<https://www.ukclimbing.com/logbook/crags/dunmore_culdaff-8761/>
ukclimbing.com<https://www.ukclimbing.com/logbook/crags/dunmore_culdaff-8761/>


Ukc approach notes are a bit vague sometimes but anything is better than nothing i suppose

Sent from my iPhone

On 4 Mar 2025, at 02:34, Rory Sullivan ***@***.***> wrote:

﻿

Hi, I can definitely add those locations, would you mind sending on links to where they are? I'm actually not familiar with Rocklands quarry or Culldaff.

As for adding a what 3 words location, I feel that a simple location pin is not necessarily enough information for how to find a crag and providing a full approach description is out of scope for Dry Rock. However I would be open to providing a link to the UKC (or similar) page for each location. Do you think that would help?

**Comment by damienledger1** [comment]
Maybe just a link to ukc log for each crag then would give everyone all the crag info approach and routes without telling them exactly how to get there keep the purists happy
Sent from my iPhone

On 4 Mar 2025, at 08:52, damien st ledger ***@***.***> wrote:

﻿
<http://wiki.climbing.ie/index.php?title=Rocklands>
<Rocklands2.jpg>
Rocklands - Irish Climbing Wiki<http://wiki.climbing.ie/index.php?title=Rocklands>
wiki.climbing.ie<http://wiki.climbing.ie/index.php?title=Rocklands>

<https://www.ukclimbing.com/logbook/crags/dunmore_culdaff-8761/>
<98364.jpeg>
Dunmore, Culdaff<https://www.ukclimbing.com/logbook/crags/dunmore_culdaff-8761/>
ukclimbing.com<https://www.ukclimbing.com/logbook/crags/dunmore_culdaff-8761/>


Ukc approach notes are a bit vague sometimes but anything is better than nothing i suppose

Sent from my iPhone

On 4 Mar 2025, at 02:34, Rory Sullivan ***@***.***> wrote:

﻿

Hi, I can definitely add those locations, would you mind sending on links to where they are? I'm actually not familiar with Rocklands quarry or Culldaff.

As for adding a what 3 words location, I feel that a simple location pin is not necessarily enough information for how to find a crag and providing a full approach description is out of scope for Dry Rock. However I would be open to providing a link to the UKC (or similar) page for each location. Do you think that would help?

**Reply by Rory-Sullivan** [reply]
I have just made a release that adds Dunmore and Rocklands as locations.

I will have a look at adding UKC links over the next few days.

**Comment by damienledger1** [comment]
Amazing work
Sent from my iPhone

On 18 Mar 2025, at 15:52, Rory Sullivan ***@***.***> wrote:

﻿

I have just made a release that adds Dunmore and Rocklands as locations.

I will have a look at adding UKC links over the next few days.

**Comment by vadimdunne** [comment]
Hey! Super cool project. Do you think you could add the Scalp as a location? At the moment, I'm using Met Éireann - Enniskerry to check.
